spec/controllers/users/phone_confirmation_controller_spec.rb

```diff
@@ -42,14 +42,17 @@
         end
 
         it 're-sends existing code' do
-          expect(SmsOtpSenderJob).to receive(:perform_later).
-            with(
-              code: '1234',
-              phone: '+1 (555) 555-5555',
-              otp_created_at: controller.current_user.direct_otp_sent_at.to_s
-            )
-
-          get :send_code
+          now = Time.zone.now
+          Timecop.freeze(now) do
+            expect(SmsOtpSenderJob).to receive(:perform_later).
+              with(
+                code: '1234',
+                phone: '+1 (555) 555-5555',
+                otp_created_at: now.to_s
+              )
+
+            get :send_code
+          end
```

Comment:
In this case, where we are mocking and stubbing a class, I don't think we have any choice but to write a test that knows about the args needed to initialize `SmsOtpSenderJob`. That being said, if we add `now.to_s` as a default arg rather than requiring it, we wouldn't need `Timecop` here.